packages/integration-tests/projects/suite-web/tests/suite/new-account-message.test.ts

```diff
@@ -0,0 +1,42 @@
+// @group:suite
+// @retry=2
+
+describe('New accounts', () => {
+        beforeEach(() => {
+             // Launches Trezor emulator
+            cy.task('startEmu', { wipe: true });
+            // Initializes Trezor (random seed)
+            cy.task('setupEmu', { needs_backup: false });
+            cy.task('startBridge');
+            cy.viewport(1024, 768).resetDb();
+            cy.prefixedVisit('/');
+            cy.passThroughInitialRun();
+            cy.discoveryShouldFinish();
+        });
+    
+
+    it('Goes to accounts and verifies that the "New default Accounts natively.." is displayed:', () => {
+        cy.prefixedVisit('/accounts/');
+        cy.getTestElement('@accounts/empty-account/receive');
+        cy.getTestElement('@accounts/empty-account/default-native-account/close');
+        cy.contains('New default Accounts natively in Trezor Suite!');
+        // Clicks on Got it!
+        cy.contains('New default Accounts natively in Trezor Suite!');
+        cy.getTestElement('@accounts/empty-account/default-native-account/close').click({
+            scrollBehavior: 'bottom',
+        });;
+        // Makes sure it is gone.
+        cy.getTestElement('@accounts/empty-account/receive');
+        cy.getTestElement('@accounts/empty-account/buy');
+        cy.getTestElement('@accounts/empty-account/default-native-account/close').should('not.exist');
+        cy.contains('New default Accounts natively in Trezor Suite!').should('not.exist');
+        // Reload page.
+        cy.reload();
+        cy.getTestElement('@device-acquire', {timeout: 3000}).click();
```

Comment:
unacquired device. This is something I would like to test in a separate test as this is not a happy path. Unacquired device happens because of doing full reload on line 19. Once you will resolve the first comment you will not need this line 35 at all. 

But anyway, great idea to test unacquired device by doing reload during discovery!

Reply:
cb0a3c4762e4b4db44b9de5a6c79d524fdc8e310 done